### PR TITLE
[ACR] Add Identity to Task [Do Not Merge]

### PR DESCRIPTION
**Pull request by jaysterp**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json [hunk 1]
```diff
@@ -1079,13 +1079,52 @@
         }
       ],
       "properties": {
+        "identity": {
```
Comment (KrisBash) [inline specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json]: There's definitely risk in adding this within an API version. If a user adds it in a recent client/portal, and a user with a legacy client does an update to the same resource (having no identity property as it was ), what will you do to the identity property? The proper behavior would be to remove `identity` value, but that is unlikely what is intended. By adding this in a new API version, you can have more intelligent handling of downlevel clients.

**Review by KrisBash — CHANGES_REQUESTED** [review]
please see comment regarding API version

**Comment by praries880** [comment]
@jaysterp  ARM requested changes... can you kindly resolve those?

**Comment by jaysterp** [comment]
Abandoning this PR as this feature will be included in a newer API version.

**Pull request closed without merging**